.vscode/cspell.json

```diff
@@ -840,6 +840,8 @@
     {
       "filename": "/sdk/ai/**",
       "words": [
+        "GENAI",
+        "OTEL",
```

Comment:
Probably not needed anymore?

```suggestion
```

Reply:
OTEL is still needed (e.g. OTEL_SCHEMA_URL const), but we can remove GENAI now (updated).